config-templates/aws/aws-efs-csi-driver/1.3.1/storage-class-parameters.json

```diff
@@ -27,8 +31,10 @@
 	"templateKeyName": [
 		"fileSystemId"
 	],
+	"regex": "^(?:fs-[a-f0-9]{8,17})$",
```

Comment:
It would be 8 or 17 characters. Please confirm once
https://docs.aws.amazon.com/efs/latest/ug/resource-ids.html

Reply:
Length check might not be required here, in case in future the length is changed by AWS, it will break.